package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@jaames/iro": "^5.5.2",
+    "axios": "^1.7.2",
```

Comment:
Curious about where we need this dependency

Reply:
Good catch.  This is a leftover of when I was fetching GitHub API for footer.  We can remove axios.